src/cli-apply.js

```diff
@@ -78,35 +78,11 @@ else {
     }
 }
 
-const updateRouteIds = ({host, headers, services, ...config}, state) => {
-    const updatedServices = services.map(({routes, ...service}) => {
-        const updatedRoutes = routes.map(({name, id, ...route}) => {
-            const updatedId = state.services.find((s) => s.name === service.name).routes.find((r) => r.name == name).id;
-            return {
-                name,
-                id: updatedId,
-                ...route
-            };
-        });
-        return {
-            ...service,
-            routes: updatedRoutes
-        };
-    });
-    return {
-        host,
-        headers,
-        services: updatedServices,
-        ...config
-    };
-};
-
 console.log(`Apply config to ${host}`.green);
 
 execute(config, adminApi({host, https, ignoreConsumers, cache}), screenLogger, removeRoutes, dryRun, localState)
-    .then(out => {
-        let updatedConfig = updateRouteIds(config, out);
-        updatedConfig = sanitizeConfigForSafeWrite(updatedConfig, envVarPointers);
+    .then(state => {
+        config = sanitizeConfigForSafeWrite(config, envVarPointers);
```

Comment:
Now that we don't update route ids, it's clearer that there's really no changes from kongfig apply that need to be written back to the file. I'm not deleting this in case we decide we do want that to be true at some point in the future

Reply:
this is ironic to me because we are now going from the `if` block being executed exclusively to the `else` block being executed exclusively